carelink.js

```diff
@@ -24,21 +33,27 @@ var CARELINK_SECURITY_URL = 'https://' + carelinkServerAddress + '/patient/j_sec
 var CARELINK_AFTER_LOGIN_URL = 'https://' + carelinkServerAddress + '/patient/main/login.do';
 var CARELINK_JSON_BASE_URL = 'https://' + carelinkServerAddress + '/patient/connect/ConnectViewerServlet?cpSerialNumber=NONE&msgType=last24hours&requestTime=';
 var CARELINK_LOGIN_COOKIE = '_WL_AUTHCOOKIE_JSESSIONID';
+var user_agent_string = [software.name, software.version, software.bugs.url].join(' // ');
 
 var carelinkJsonUrlNow = function () {
     return (CARELINK_EU ? CARELINKEU_JSON_BASE_URL : CARELINK_JSON_BASE_URL) + Date.now();
 };
 
 var Client = exports.Client = function (options) {
+    let requestCount = 0;
+
     if (!(this instanceof Client)) {
         return new Client(arguments[0]);
     }
 
     axiosCookieJarSupport(axios);
-    var cookieJar = new tough.CookieJar();
```

Comment:
Agree the cookie jar is now initiated two times on consecutive rows. However, if you remove the var `cookieJar` here, it is undefined in row 281...

Reply:
I believe this was due to a mistake on my end while resolving conflicts and should be fixed.